app/pb_kits/playbook/pb_currency/_currency.jsx

```diff
@@ -30,26 +22,43 @@ const sizes = {
 
 const Currency = ({
   align = 'left',
-  currencySymbol = '$',
+  amount,
+  className,
   label = '',
   separator = '.',
   size = 'sm',
-  units = '00',
-  value,
-}: CurrencyProps) => (
-  <div className={buildCss('pb_currency_kit', align)}>
-    <Caption>{label}</Caption>
-    <div className="pb_currency_wrapper">
-      <Body className={symbolCSS(currencySymbol)}>{currencySymbol}</Body>
-      <Title
-          className="pb_currency_value"
-          size={sizes[size]}
-      >
-        {`${value}${separator}`}
-      </Title>
-      <Body className="unit">{units}</Body>
+  symbol = '$',
+}: CurrencyProps) => {
+  const [whole, decimal = '00'] = amount.split(separator)
```

Comment:
Are there any plans to introduce JS testing? I'd love to add coverage to this kind of logic ☝️ 

If not, I may get to setting up a test framework (mocha/chai/sinon or Jest) in Playbook to start introducing React test coverage as well, thoughts?

Reply:
There are plans to get that stuff setup in PB. I would like to throw out a big vote for React-Testing-Library. It couples really well with Jest, which i also think is a good test runner for us. I think this combo is a solid route to go in Playbook because it supports a lot of modern syntax including simple testing for hooks. https://github.com/testing-library/react-testing-library 

Have you checked out that one at all, @diego? Kent Dodds uses it in all of his courses here as well: testingjavascript.com (here's a quick video of him talking a little about react-testing-library-- i've got it marked at the 15min marker for you: https://www.youtube.com/watch?v=hZFgyoImTx0&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u%3Ft%3D15m1s)

cc: @thestephenmarshall @jasoncypret 

A few more links I'm investigating lately as well: 
https://www.reddit.com/r/reactjs/comments/d9oux9/test_driven_development_tdd_with_react_react/

https://www.reddit.com/r/reactjs/comments/d9oux9/test_driven_development_tdd_with_react_react/f1ln5ey/?context=8&depth=9

Reply:
> Have you checked out that one at all, @diego?

Yes, that's what I usually use on my projects nowadays.

I would also consider https://www.cypress.io for end-2-end tests. I'm quite sure we could use it to test both Rails and React code (against https://playbook.powerapp.cloud itself), but cypress really shines when testing more interactive/async widgets which could be handy for more complex React components. Plus `Cypress` is almost `Cypret`, right @jasoncypret? 😆